portal/src/texts/profile/colors.md

```diff
@@ -1,91 +1,93 @@
+**Funksjonsfargene** tar vi i bruk på digitale flater, når vi trenger flere valører enn det som finnes i hoved- og støttefargene. De brukes også når vi skal oppfylle krav om tilgjengelighet på en god måte (for eksempel røde feilmeldinger mot lys bakgrunn).
 
 
 ### Farger i digitale løsninger
 
-I digitale løsninger benytter vi oss av våre hovedfarger og funksjonsfarger. Støttefargene er i utgangspunkt forbeholdt illustrasjoner og grafikk men kan veldig sparsomt brukes til å fremheve informasjon i løsningene våre. Spør alltid om det kan heller løses via våre hovedfarger.
+I digitale løsninger bruker vi hovedfargene og funksjonsfargene. Støttefargene reserverer vi til illustrasjoner og grafikk, men vi kan bruke dem sparsomt til å fremheve informasjon i løsningene våre. Spør deg selv alltid om du heller kan løse det ved å bruke hovedfargene.
 
 
 ### Tilgjengelighet
 
-Tilgjengelighet er det viktigste å tenke på når du benytter fargene i digitale løsninger. Vi skal alltid etterkomme kravene til universell utforming, som for farger vil bety [AA-kravet for kontrast](link). Kontrastkravet gjelder ikke bare tekst på bakgrunn, men også komponentenes kontrast mot sin bakgrunn når den er av betydning for å forstå funksjonaliteten (e.g. ikke bare tekst på knappebakgrunnen, men også knappebakgrunnen mot sidens bakgrunnsfarge).
+Tilgjengelighet er det viktigste å tenke på når vi bruker fargene i digitale løsninger. Vi skal alltid følge kravene til universell utforming. For farger betyr det [AA-kravet for kontrast](lenke). Kontrastkravet gjelder ikke bare tekst på bakgrunn, men også komponentenes kontrast mot sin bakgrunn, når det har betydning for å forstå funksjonaliteten 
+I tillegg til å tenke på tekst mot knappebakgrunnen, må vi for eksempel også tenke på knappebakgrunnen mot sidens bakgrunnsfarge.
 
-Tekstfargene våre er helhvit eller svart. Referer til fargebeskrivelsene for å vite hvilken farge du skal benytte mot hvilken bakgrunn for å ivareta kontrastkravet.
+Tekstfargene våre er helhvit eller svart. Se på fargebeskrivelsene for å finne ut hvilken farge du skal bruke mot hvilken bakgrunn, for å overholde kontrastkravet.
 
 #### Funksjonsfarger
-Funksjonsfargene rød, blå og grønn brukes for å gi informasjon om systemets tilstand. Rødt for feilmeldinger, grønt for suksess og blå for nøytral informasjon. Disse fargene skal kun benyttes til dette formålet og ikke i illustrasjoner eller grafikker (dette gjelder ikke for de støttefargene som også benyttes i samme funksjonsøyemed).
-
+Vi bruker funksjonsfargene rød, blå og grønn til å gi informasjon om systemets tilstand. Rødt er til feilmeldinger, grønt til en vellykket operasjon og blå til nøytral informasjon. Vi skal bare bruke disse fargene til dette formålet, og ikke i illustrasjoner eller grafikk. Vi kan bruke alle støttefargene i illustrasjoner og grafikk.
```

Comment:
Jeg syntes det er uklart når vi skriver "Vi kan bruke alle støttefargene i illustrasjoner og grafikk." siden vi ikke har referert til støttefarger tidligere i seksjonen. Jeg ville vurdert å kutte det siden fargene som er støttefarger men kan brukes som funksjonsfarger kun er listet som støttefarger lenger ned.

Reply:
Kutt det du, siden du argumenterte for det selv :-)